src/i18n/locales/en/about_us.json

```diff
@@ -2,7 +2,7 @@
   "en": {
     "heading1": "Um Mosteiro",
     "heading2": "Uma família",
-    "heading3": "Nove gerações",
+    "heading3": "Dez gerações",
```

Comment:
Just in case someone is checking the family tree, hey? 😜

Reply:
😉

Reply:
Can you also change the font size by 2 pixels? 😜